python/pfs/drp/stella/reduceArcTask.py

```diff
@@ -149,7 +138,6 @@ def run(self, expRefList, butler, wLenFile=None, lineList=None, immediate=True):
                 spec.setITrace(iTraces[i])
                 self.log.info('flatFiberTraceSet.getFiberTrace(%d).getITrace() = %d, spec.getITrace() = %d' %(i,flatFiberTraceSet.getFiberTrace(i).getITrace(), spec.getITrace()))
                 specSpec = spec.getSpectrum()
-                self.log.info('yCenters.shape = %d' % yCenters.shape)
```

Comment:
Why? I think the log message was probably redundant (I guess it logged the same thing for every loop iteration), but it now seems to be gone entirely (ie not even logged once) without explanation. If you just don't want it any more, I'd have this on a separate commit with a message like "Removed redundant log message".